app/services/users/profile_contexts/foa.rb

```
@@ -21,6 +21,10 @@ def copy_instance_allowed?
     true
   end
 
+  def new_instance_allowed?
+    user.groups.include?('v2-profile-instance-edit')
```

RuboCop (1.69.1)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)